pkg/webclient/cloudproviders/genqlient.yaml

```diff
@@ -1,6 +1,6 @@
 # For full documentation see:
 # https://github.com/Khan/genqlient/blob/main/docs/genqlient.yaml
-schema: ../../../api/graphql/schema.graphql
```

Comment:
Isn't a standard in graphql to have the file where it was before this change?

Reply:
Yes, if it were an API for a service. In this case, the API is to generate a web client. It does not make sense to be in a different folder than the client itself.